scripts/gen_help_html.lua

```diff
@@ -36,9 +36,14 @@ local spell_dict = {
   VimL = 'Vimscript',
   vimL = 'Vimscript',
   viml = 'Vimscript',
+  ['tree-sitter'] = 'treesitter',
+  ['Tree-sitter'] = 'Treesitter',
 }
+--- specify the list of keywords to ignore (i.e. allow), or true to disable spell check completely.
+--- @type table<string, true|string[]>
 local spell_ignore_files = {
-  ['backers.txt'] = 'true',
+  ['backers.txt'] = true,
+  ['news.txt'] = { 'tree-sitter' }, -- in news, may refer to the upstream "tree-sitter" library
```

Comment:
#27067 (news.txt)

> Treesitter highlight groups have been renamed to be more in line with
upstream **tree-sitter** and Helix to make it easier to share queries. The full
list is documented in |treesitter-highlight-groups|.